CHANGELOG.md

```diff
@@ -1,3 +1,117 @@
+# 100.0.0
+
+In this major release we made improvements to Ink. As well as improvements including customization Dialogs presentation animation,
+Flexible Header behavior around safe area. Also, we addressed bugs in Snackbar and TextFields.
```

Comment:
tip: In this major release we made a breaking change improvement to Ink’s inkColor API.

Reply:
Done